geex-core/src/main/java/nl/tudelft/context/controller/locator/LocatorController.java

```diff
@@ -140,6 +154,11 @@ private void setPosition() {
             locatorIndicator.setWidth((max - min) * scale);
         });
 
+        if (width != locator.getWidth()) {
```

Comment:
Best solved with a property.
You call a function in an other controller that calls itself again.
So you don't need the graph controller dependency in this locator yet.